File: app/src/main/java/com.caique.everis.testeandroid/model/AccountInfo.java
package com.caique.everis.testeandroid.model;

import com.google.gson.annotations.SerializedName;

public class AccountInfo {

    @SerializedName("title")
    private String title;

    @SerializedName("desc")
    private String desc;

    @SerializedName("date")
    private String date;

    @SerializedName("value")
    private Double value;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Double getValue() {
        return value;
    }

    public void setValue(Double value) {
        this.value = value;
    }
}

File: teste/app/src/main/java/com/example/ilealnod/SantanderProjectKotlin/Todos/Dependencias/AccountInfo.kt
package com.example.ilealnod.SantanderProjectKotlin.Todos.Dependencias

import com.google.gson.annotations.SerializedName

class AccountInfo {

    //Dependencia de Informações da conta

    @SerializedName("title")
    var title: String? = null
    @SerializedName("desc")
    var desc: String? = null
    @SerializedName("date")
    var date: String? = null
    @SerializedName("value")
    var value: Double? = null

}
